OneCommunityGlobal/HighestGoodNetworkApp
# weiyao_permissionManagement_addRoles

# Description
After the owner gives the volunteer permission (add roles), the volunteer cannot create new role because all the buttons are disabled. After the fix, they should be enabled.

## Related PRS (if any):
- weiyao_permissionManagement_addRoles
- For backend use development branch


## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as owner
5. Go to permission management and give a volunteer permission “add roles”
6. Go to that volunteer account
7. input a test role name
8. see if you can see enabled buttons, check several
9. see if you can successfully create a role by seeing the pop-up

Note:
In order to see if the created role, I think this pr 2386 has to be merged first by fixing the “not showing” bug.
However, you should at least see the success popup demonstrating you successfully created a new role.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2386

## Screenshots or videos of changes:
https://www.loom.com/share/3f6bc745b86d4cca9b19ec4b928829a5?sid=2949f3a8-9fa7-4fca-b471-df068bc99f59
<img width="997" alt="Pasted Graphic 2" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/96925606/bffb48cd-1fe0-4eaf-bc8e-869a3aed626f">
￼
Before:
<img width="805" alt="User Management" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/96925606/b20e4aae-1399-47c9-9c6e-7b8ad671dd7d">

after:
All disabled buttons should be enabled and role created successfully 
￼
<img width="448" alt="Pasted Graphic 1" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/96925606/823171ca-67f1-4f56-b7df-0247ed6b7137">
